src/FormTaxonomies/resources/form-builder/form-categories.tsx

```diff
@@ -0,0 +1,54 @@
+import {getAvailableFormCategories, getInitialFormCategories} from "./windowData";
+import {buildTermsTree} from "./utils/terms";
+import {CheckboxControl} from "@wordpress/components";
+import {decodeEntities} from "@wordpress/html-entities";
+
+
+const FormCategorySetting = ({settings, setSettings}) => {
+    const {
+        formCategories = getInitialFormCategories(),
+    } = settings;
```

Comment:
Again, if we are bypassing our internal form settings and treating this as an extension than this would be the approach to take:

```suggestion
    const {
        formCategories = getInitialFormCategories(),
    } = = {...giveTaxonomySettings, ...settings};
```